### PR TITLE
4.8.10 async analysis

### PR DESCRIPTION
**Pull request by karthik-tarento**

### CODE REVIEW

File: all-actors/src/main/java/org/sunbird/notification/handler/EmailNotificationHandler.java [hunk 1]
```diff
@@ -53,10 +54,12 @@ public Response sendNotification(NotificationV2Request notificationRequest, bool
     }
 
     private NotificationRequest createNotificationObj(NotificationV2Request notificationRequest, Map<String,Object> templateConfig) {
-
+        logger.info("In Create V1 Notification object");
```
Comment (karthik-tarento) [inline all-actors/src/main/java/org/sunbird/notification/handler/EmailNotificationHandler.java]: Remove unnecessary logs

File: all-actors/src/main/java/org/sunbird/notification/handler/EmailNotificationHandler.java [hunk 2]
```diff
@@ -53,10 +54,12 @@ public Response sendNotification(NotificationV2Request notificationRequest, bool
...
         NotificationRequest notification = new NotificationRequest();
         notification.setIds(notificationRequest.getIds());
         notification.setMode(DeliveryMode.email.name());
+        logger.info("cc email value is : " + notificationRequest.getCopyEmail());
```
Comment (karthik-tarento) [inline all-actors/src/main/java/org/sunbird/notification/handler/EmailNotificationHandler.java]: Remove this unnecessary logs

File: all-actors/src/main/java/org/sunbird/notification/handler/EmailNotificationHandler.java [hunk 3]
```diff
@@ -67,6 +70,11 @@ private NotificationRequest createNotificationObj(NotificationV2Request notifica
         template.setParams(jsonNode);
         notification.setTemplate(template);
         notification.setDeliveryType(NotificationRouter.DeliveryType.message.name());
+        try {
+            logger.info("Notification Request is : " + mapper.writeValueAsString(notification));
+        } catch (JsonProcessingException e) {
+            logger.info("Exception occurred while writing notification as json object");
+        }
```
Comment (karthik-tarento) [inline all-actors/src/main/java/org/sunbird/notification/handler/EmailNotificationHandler.java]: Remove this unnecessary logs

File: notification-sdk/src/main/java/org/sunbird/notification/email/Email.java [hunk 4]
```diff
@@ -275,7 +272,7 @@ private boolean sendEmail(Session session, MimeMessage message) {
     try {
       transport = getTransportClient(session);
       transport.sendMessage(message, message.getAllRecipients());
-        logger.info(message.getContent().toString());
+      logger.info(message.getContent().toString());
```
Comment (karthik-tarento) [inline notification-sdk/src/main/java/org/sunbird/notification/email/Email.java]: Remove this unnecessary logs

File: all-actors/src/main/java/org/sunbird/notification/handler/EmailNotificationHandler.java (hunk 3 above)
Comment (ravisaurav-tarento) [reply all-actors/src/main/java/org/sunbird/notification/handler/EmailNotificationHandler.java]: This is done

File: notification-sdk/src/main/java/org/sunbird/notification/email/Email.java (hunk 4 above)
Comment (ravisaurav-tarento) [reply notification-sdk/src/main/java/org/sunbird/notification/email/Email.java]: This was pre-existing Karthik

File: all-actors/src/main/java/org/sunbird/notification/handler/EmailNotificationHandler.java (hunk 2 above)
Comment (ravisaurav-tarento) [reply all-actors/src/main/java/org/sunbird/notification/handler/EmailNotificationHandler.java]: Done

File: all-actors/src/main/java/org/sunbird/notification/handler/EmailNotificationHandler.java (hunk 1 above)
Comment (ravisaurav-tarento) [reply all-actors/src/main/java/org/sunbird/notification/handler/EmailNotificationHandler.java]: This is done

**Pull request merged**